src/components/DeviceButton.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <li>
+    <a class="change-device"
+      :class="[
+        deviceTrigger,
+        device == deviceTrigger ? 'active': ''
+      ]"
+      v-on:click="displayOn(deviceTrigger)"
```

Comment:
Next in the project, you use some directives with shorthand practice, Vue recommend to use one or the other (shorthand or complete directive), I would then recommend you to use @click or v-on everywhere but not mix them in the entire project!

Reply:
I created https://github.com/PrestaShop/live-demo-devices/issues/15, so we can move forward with this PR without forgetting your comment.